### PR TITLE
Redesign site and add code of conduct

### PR DESCRIPTION
**Pull request by iamtommcc**
Implementing the new synthwave-style brand, and adding a code of conduct!

### CODE REVIEW

File: pages/_error.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ class Error extends React.Component {
     return (
       <>
         <PageHead />
-
```
Comment (loklaan) [inline pages/_error.js]: FYI this isn't reflected yet in deployments due to `https://github.com/zeit/now-builders/issues/234`

File: readme.md [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
 
-Clone down this repo, and start hacking away.
+Clone down this repo, run `yarn` and start hacking away.
 
-It's plain ol' HTML & CSS right now, so you can preview it in any browser without start a dev server.
+`yarn dev` to start up the dev server, `yarn build` to build
 
 ## Legal
 
-Copyright © 2018 ReactBris
```
Comment (loklaan) [inline readme.md]: tyty

File: static/styles.css [hunk 3]
```diff
@@ -33,79 +42,44 @@ body {
...
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  margin: auto;
+  opacity: 0;
+  z-index: 999;
+  animation: fadeInAndGrow 2s ease-out forwards;
+  animation-delay: 300ms;
+  margin-top: 250px;
```
Comment (loklaan) [inline static/styles.css]: When viewing the site on a macbook viewport, this top margin pushed the logo down too much. 😢 It also has a higher z index than the navigation, so it blocks click.

* Make position of logo responsive
* Remove pointer-events

File: static/styles.css [hunk 4]
```diff
@@ -33,79 +42,44 @@ body {
...
+  color: white;
+  background: rgba(9, 12, 37, 0.7);
+  padding: 60px;
   width: 100%;
   text-align: center;
 }
 
 .page-error h1 {
-  font-size: 20rem;
+  font-size: 40px;
```
Comment (loklaan) [inline static/styles.css]: y u h8 rem

File: static/styles.css [hunk 5]
```diff
@@ -181,21 +148,109 @@ a {
...
+    opacity: 1;
+  }
+
+  100% {
+    transform: scale(1);
+    opacity: 1;
+  }
+}
+
+.skyline {
```
Comment (loklaan) [inline static/styles.css]: This is cool! It's broken though, at least on laptop viewports.

* Make responsive

File: static/styles.css [hunk 6]
```diff
@@ -204,6 +259,100 @@ a {
...
+}
+
+.content-inner {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 60px 80px 80px 80px;
+}
+
+.content::before {
+  content: "" "";
```
Comment (loklaan) [inline static/styles.css]: I've never seen this before, what does it mean?

**Comment by loklaan** [comment]
Hey also for brand, I'd prefer if we make **primary** use of [the](https://reactbris.org) [stuff](
https://www.figma.com/file/P9jfvdcf8HojZ4zUW71mWFlT/ReactBris?node-id=67%3A642) [I've](https://www.meetup.com/reactbris/) [done](https://twitter.com/reactbris) for 2019, or atleast until I'm outa-ere. I was thinking this synth-style would be complementary, e.g. for transient collateral like what is seen on Screens at event's etc. What's your thoughts here?

**Comment by iamtommcc** [comment]
Thanks for the looksey @loklaan - and thanks for the responsiveness check, went a bit half-assed there!

Re. the brand: all good if you'd like to stick with the stuff we've already got.

If we do stick with the current style I don't think it makes much sense to mix in this synthwave style anywhere. Probably just best to go 100% with the one style IMO. Logic being that the brand is only expressed in a few small places... too much variance would dilute the brand and make it feel like there's _no_ brand.

With that in mind, I'll close this one for now :+1:

**Pull request closed without merging**